Dockerfile

```diff
@@ -30,18 +25,29 @@ RUN set -x \
     && docker-php-ext-install odbc \
     && docker-php-source delete
 
-COPY --from=0 /tmp/snowflake-odbc.deb /tmp/snowflake-odbc.deb
-RUN dpkg -i /tmp/snowflake-odbc.deb
-ADD ./driver/simba.snowflake.ini /usr/lib/snowflake/odbc/lib/simba.snowflake.ini
+#snoflake download + verify package
+ADD driver/snowflake-policy.pol /etc/debsig/policies/$SNOWFLAKE_GPG_KEY/generic.pol
```

Comment:
je to drobnost - ale lokální soubory by se měly kopírovat přes `COPY`.

ADD dělá věci navíc (extrahuje tary a umí stahovat z HTTP). 

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

Reply:
je to v jiném PR ale vycházel jsem z něj - doplnil jsem to do description PR